src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+import static duke.constant.Message.CLOSE_BRACKET;
+import static duke.constant.Message.OPEN_BRACKET;
+import static duke.constant.TaskConstant.PREFIX_EVENT;
+
+/**
+ * A class represents for an event.
+ */
+public class Event extends Task{
+    protected String at;
```

Comment:
Perhaps a slightly more intuitive variable name here?
The variable "at" is not as descriptive as it could be! Perhaps you could use a variable like "details" instead if it makes sense!